plugins/pixart-rf/fu-pxi-device-common.c

```diff
@@ -18,27 +18,28 @@ fu_pxi_device_set_feature (FuDevice *self,
                            guint datasz,
                            GError **error)
 {
-    fu_common_dump_raw (G_LOG_DOMAIN, "SetFeature", data, datasz);
-    return fu_udev_device_ioctl (FU_UDEV_DEVICE (self),
-                                 HIDIOCSFEATURE(datasz), (guint8 *) data,
-                                 NULL, error);
+        fu_common_dump_raw (G_LOG_DOMAIN, "SetFeature", data, datasz);
+        return fu_udev_device_ioctl (FU_UDEV_DEVICE (self),
+                                     HIDIOCSFEATURE(datasz), (guint8 *) data,
+                                     NULL, error);
 }
 
 gboolean
 fu_pxi_device_get_hid_raw_info(FuDevice *self,
                                struct hidraw_devinfo *info,
                                GError **error)
 {
-    if (!fu_udev_device_ioctl (FU_UDEV_DEVICE (self),
-                               HIDIOCGRAWINFO, (guint8 *) info,
-                               NULL, error))
-        return FALSE;
+        if (!fu_udev_device_ioctl (FU_UDEV_DEVICE (self),
+                                   HIDIOCGRAWINFO, (guint8 *) info,
+                                   NULL, error)) {
+                return FALSE;
+        }
 
-    g_debug("bustype: %d",(gint32)info->bustype);
-    g_debug("vendor: 0x %04hx", info->vendor);
-    g_debug("product: 0x%04hx", info->product);
+        g_debug("bustype: %d",(gint32)info->bustype);
+        g_debug("vendor: 0x %04hx", info->vendor);
+        g_debug("product: 0x%04hx", info->product);
```

Comment:
This needs to be one tab char, not 8 spaces. You also always need a space before the left `(`

Reply:
I got it thank !